std_msgs/Float32MultiArray y_ann
std_msgs/Float32MultiArray y_force
std_msgs/Float32MultiArray y_err
std_msgs/Float32MultiArray y_target

std_msgs/Float32MultiArray res_states
std_msgs/Float32MultiArray res_in

float32 wout_sum
